### DISCUSSION TITLE
Install Experience User Utilities Extensions 1.7

### DISCUSSION THREAD

**Discussion by juergensens** [ildiavolorosso/ExperienceUserUtilities#10] [General]
Problem:
1. (Experience User Utilities 1.6) A required package is missing
Package "Experience User Utilities", Version 1.6 or later must be installed first.

How can I get the former version of the package?

Thanks for the answer

**Comment by juergensens** [comment]
Got the right link for v1.6 over your repo here. When searching on Appexchange, I already get on the link v1.7

**Reply by ildiavolorosso** [reply]
Thanks for letting me know. So you installed the base package (1.7) first then the extensions and it gave you that message?

**Reply by juergensens** [reply]
Yes, I tried it again and it could be that this was my issue :)
